getting_started/index.adoc

```diff
@@ -62,3 +87,79 @@ For the {product-title} (Next Gen) Developer Preview offering, consult the
 xref:../getting_started/devpreview_faq.adoc#getting-started-devpreview-faq[Developer Preview FAQ] for details
 around current usage considerations during this preview phase.
 endif::openshift-online[]
+
+ifdef::openshift-origin,openshift-enterprise[]
+
+If you do not already have {product-title} installed, the following sections
+will help guide you through the installation process:
+
+* xref:../getting_started/index.adoc#install-openshift[Installing {product-title}]
+* xref:../getting_started/index.adoc#interact-with-openshift[Interact with {product-title}]
+* xref:../getting_started/index.adoc#roles-and-authentication[Understand Roles and Authentication]
+* xref:../getting_started/index.adoc#deploy-router[Deploy the OpenShift Router]
+* xref:../getting_started/beyond_the_basics.adoc#deploy-docker-registry[a Docker Registry]
+
+[[install-openshift]]
+== Install {product-title}
```

Comment:
@mburke5678 - hey Michael, yeah the intent is to guide users with a non production instance.